homeassistant/components/bmw_connected_drive/__init__.py

```diff
@@ -20,7 +20,7 @@
 
 DOMAIN = 'bmw_connected_drive'
 CONF_REGION = 'region'
-
+CONF_VIN = 'vin'
```

Comment:
`CONF_` constants are only used in config schemas. Here you want to use `ATTR_` in the service schema.

Reply:
fixed